app/imports/startup/slingshot.js

```diff
@@ -0,0 +1,39 @@
+/* globals Slingshot Random */
+import { Meteor } from 'meteor/meteor';
+import { Random } from 'meteor/random';
+
+Slingshot.fileRestrictions('imageUpload', {
+  allowedFileTypes: ['image/png', 'image/jpeg', 'image/gif'],
```

Comment:
i rly dont know, specially since iOS could open the gallery. but this types are right no?